slue_toolkit/prepare/prepare_voxpopuli.py

```diff
@@ -94,6 +95,23 @@ def create_manifest(
                         )
                         print(wrd_str, file=f_wrd)
                         print(ltr_str, file=f_ltr)
+        try:
```

Comment:
@fwu-asapp Why should there be a file at `../{split}.tsv'? And if there is one, why not move it to the `manifest_dir` instead of symlink?

Reply:
It should also be there for ASR tasks.

Reply:
Sorry, I misunderstood your question. The `../{split}.tsv` is means `manifest/slue-voxpopuli/{split}.tsv` because the symbolic link is in `manifest/slue-voxpopuli/e2e_ner`. Starting from this folder, `..` means `manifest/slue-voxpopuli`.
There is nothing to do with `slue_toolkit/prepare`, the relative path of the symbolic link is based on where the link is, not where the code is.

Reply:
My original answer is that `manifest/slue-voxpopuli/{split}.tsv` is used for ASR and should be kept there as well instead of deleting it and only having `manifest/slue-voxpopuli/e2e_ner/{split}.tsv`